lib/screens/create_region_screens/viewmodels/create_region_state.dart

```diff
@@ -6,36 +6,46 @@ class CreateRegionState extends Equatable {
   final PageState pageState;
   final PageCommand? pageCommand;
   final CreateRegionScreen createRegionsScreens;
+  final File? file;
+  final bool isUploadImageNextAvailable;
```

Comment:
Do we need this as a state? 
why not a getter?
`bool get isUploadImageNextAvailable => file!=null;`

Reply:
Also I think has more sense isUploadImageNextButtonEnabled

Reply:
I dont like logic on state unless is necessary, the name I will change on another pr cause I have all other buttons as available

Reply:
Im fine with your choice.
Its just I don't think this var falls as bussines logic its more a UI logic to determine when a button is enabled. not realy complex logic too.

Reply:
will think it over, still have like 4 more prs for these screens haha